.gitignore

```diff
@@ -14,4 +14,10 @@
 /coverage/*
 /libpeerconnection.log
 npm-debug.log*
+yarn-error.log
```

Comment:
Is this just from ember-cli's blueprint? Because we're not using Yarn here

Reply:
Yes, it's part of the default blueprint